src/EventListener/LogRevisionsListener.php

```diff
@@ -451,7 +452,7 @@ private function saveRevisionEntityData(ClassMetadata $class, array $entityData,
         }
 
         if ($class->isInheritanceTypeJoined() && $class->name !== $class->rootEntityName) {
-            $entityData[$class->discriminatorColumn['name']] = $class->discriminatorValue;
+            $entityData[(string) $class->discriminatorColumn['name']] = $class->discriminatorValue;
```

Comment:
I added phpdoc on orm but it's a quickfix

Reply:
will it be released with a patch on 2.13? afaik they released 2.13.1 a few hours ago

Reply:
It will be with https://github.com/doctrine/orm/pull/9983 ; but phpstan will report a useless cast so we wont forget to fix it.